lib/commands/install.js

```diff
@@ -13,7 +13,15 @@ module.exports = Command.extend({
     { name: 'save', type: Boolean, default: false, aliases: ['S'] },
     { name: 'save-dev', type: Boolean, default: true, aliases: ['D'] },
     { name: 'save-exact', type: Boolean, default: false, aliases: ['E', 'exact'] },
-    { name: 'yarn', type: Boolean, description: 'Use --yarn to enforce yarn usage, or --no-yarn to enforce npm' }, // no default means use yarn if the blueprint has a yarn.lock
+    {
+      name: 'package-manager',
+      type: String,
```

Comment:
I think this works, right?

```suggestion
      type: ['yarn','npm','pnpm'],
```

Basically, we don't want to allow "any" string; only specfic ones.